cxx/isce3/geocode/GeocodeCov.cpp

```diff
@@ -480,7 +484,7 @@ void Geocode<T>::geocodeInterp(
             else
                 rtc_memory_mode = isce3::core::MemoryModeBlocksY::MultipleBlocksY;
 
-            if (out_geo_rtc_gamma0_to_sigma0 != nullptr) {
+            if (out_geo_rtc_gamma0_to_sigma0 != nullptr && output_rtc_sigma == nullptr) {
                 std::string vsimem_ref = (
                     "/vsimem/" + getTempString("geocode_cov_areaproj_rtc_sigma0"));
```

Comment:
Looks like you might've missed this change when copying from the internal repo.

```suggestion
                    "/vsimem/" + getTempString("geocode_cov_interp_rtc_sigma0"));
```

Reply:
Good catch. I forgot to run `pull` the changes before pushing them to the public repo. Thanks!